Foundation/NSUUID.swift

```diff
@@ -32,11 +32,7 @@ public class NSUUID : NSObject, NSCopying, NSSecureCoding, NSCoding {
     }
     
     public init(UUIDBytes bytes: UnsafePointer<UInt8>) {
-        if (bytes != nil) {
-            memcpy(unsafeBitCast(buffer, to: UnsafeMutablePointer<Void>.self), UnsafePointer<Void>(bytes), 16)
-        } else {
-            memset(unsafeBitCast(buffer, to: UnsafeMutablePointer<Void>.self), 0, 16)
-        }
+        memcpy(unsafeBitCast(buffer, to: UnsafeMutablePointer<Void>.self), UnsafePointer<Void>(bytes), 16)
```

Comment:
This one was strange. There's an explicit test for the removed behavior, but the Darwin API declares that the parameter is non-optional.